nautobot/docs/installation/nautobot.md

```diff
@@ -272,6 +285,9 @@ Next, connect to the name or IP of the server (as defined in `ALLOWED_HOSTS`) on
 !!! important
     Certain Nautobot features (Git repository synchronization, webhooks, jobs, etc.) depend on the presence of Nautobot's background worker process, which is not automatically started by the `runserver` command. To start it for testing purposes, you can run `nautobot-server rqworker` separately. For production use, both Nautobot and the worker should be managed by systemd rather than started manually, as described in the next section of this documentation.
 
+!!! important
+    Some unix implementations (such as CentOS 8.2.2004) have a firewall enabled by default.  If you are unable to connect to the server url on port 8000, verify the firewall status and configure to allow the appropriate connections.
```

Comment:
```suggestion
    Some Linux distributions (such as CentOS 8.2.2004) have a firewall enabled by default.  If you are unable to connect to the server URL on port 8000, verify the firewall status and configure to allow the appropriate connections.
```

Reply:
Just "CentOS" IMO thanks.

Reply:
Changed all `CentOS 8.2.xxxx` references to `CentOS`.  I wasn't sure if that behavior was specific to CentOS or specific CentOS versions, so I qualified it with the version.